### PR TITLE
Lesson 01 task

### PR DESCRIPTION
**Pull request by raydeth**

### CODE REVIEW

File: lesson_01/main.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/kyokomi/emoji/v2"
+)
+
+func main() {
+
+	// :face: not support
```
Comment (ikamornikau) [inline lesson_01/main.go:9]: What is this comment about?

File: lesson_01/main.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	emoji.Println("Hello, world, :smile: ")
+}
```
Comment (ikamornikau) [inline lesson_01/main.go]: It's considered good practice to end a file with newline

**Review by ikamornikau — APPROVED** [review]

File: lesson_01/main.go (hunk 1 above)
Comment (raydeth) [reply lesson_01/main.go:9]: > What is this comment about?

smile with type ":face:" doesn't contains in emoji pack

**Review by ikamornikau — COMMENTED** [review]
Looks good to me

**Pull request merged**